spec/models/participant_spec.rb

```diff
@@ -19,4 +20,19 @@
       end
     end
   end
+  describe 'total_score' do
+    it'participant has correct maneuver score' do
+      expected_total = participant.maneuver_score
+      if participant.onboard_judging.present?
```

Comment:
Here also, this is just duplicating the content of the existing code.

In general, if you find yourself writing `if` in a test, you're not on the right track. In a test, you structure a setup in a certain way and make assertions - there should never be conditional logic in that scenario.